### DISCUSSION TITLE
Questions regrading to phrases used in Monitorian

### DISCUSSION THREAD

**Discussion by FrzMtrsprt** [emoacht/Monitorian#377] [Q&A] [answered]
I've been trying to update the translations, but I found some of the expressions rather confusing.
1. What does `unison` mean? After a thorough search, it seems to be the name of a dual monitor stand by ZipView. I recommend that a more generic phrase would be easier to understand, `multi-monitor`, or `change brightness of multiple monitors`, for example.
2. What does `operation log` mean? Does it refer to operations of the software itself, or operations of the user? I failed find the context in the software.
3. What does `reconnection` mean? Does it mean when the software reconnects with the monitor? If so, phrases like `relaunch` or `restart` might be better.

Thanks!

**Comment by emoacht** [comment, marked as answer]
Thanks for your interest in this app.

1. Search "move in unison". It is the original description of this item. In my experience, an explanatory and longer description does not work well but a succinct and destinctive expression does.
2.  Read https://github.com/emoacht/Monitorian#operations. You don't need to translate it.
3.  Read #115. Reconnection means a monitor is connected again.

**Reply by FrzMtrsprt** [reply]
I see, thanks for your explanation.
